### PR TITLE
SignUp Page Created

### PR DESCRIPTION
**Pull request by aamrapaliwandhre**
chrome-capture (24)

### CODE REVIEW

**Comment by kgajriaAgarwal** [comment]
image
try to make it responsive 

image
you can improve the ui.

File: signUp.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                <form class="loginBox">
+                    <input placeholder="First Name" type="text" required class="signInInput loginInput" />
+                    <input placeholder="Last Name" type="text" required class="signInInput loginInput" />
+                    <input placeholder="Email" type="email" required class="signInInput loginInput" />
+                    <input placeholder="Password" type="password" required class="signInInput loginInput" />
+                    <input placeholder="Password" type="password" required class="signInInput loginInput" />
+                    <button class="loginButton" type="submit">Sign Up</button>
+                    <span class="loginForgot">Forgot Password?</span>
+                    <button class="loginRegisterButton"><a href="login.html">Already have account?</a></button>
+                </form>
```
Comment (rahulk31) [inline signUp.html]: Suggestion: Wrap the `button` with the `<a>` tag.
As of now the link gets triggered only when clicked on the text which is a bad UX.

**Comment by kgajriaAgarwal** [comment]
everything rest looks good.

**Review by rahulk31 — APPROVED** [review]
UI Suggestion: To give some margin top and bottom to "Forgot Password" text.
image

Overall, the code is well structured and you have used the classes very well.
This can be merged 👍

**Review by kgajriaAgarwal — CHANGES_REQUESTED** [review]
Kindly do the suggested changes

File: signUp.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            </div>
+        </div>
+    </div>
+
+
+
```
Comment (pradipgharge) [inline signUp.html]: remove unused extra space

File: signup.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.signup-Wrapper{
+    width: 40rem;
+    height: 100%;
+    border-color: var(--black);
+    border-width: 0.13rem;
```
Comment (pradipgharge) [inline signup.css]: can use px for less than 1 rem lengths

**Review by pradipgharge — APPROVED** [review]
looks good to merge

File: signUp.html (hunk 1 above)
Comment (aamrapaliwandhre) [reply signUp.html]: Done

**Comment by aamrapaliwandhre** [comment]
> 

done

File: signup.css (hunk 3 above)
Comment (aamrapaliwandhre) [reply signup.css]: done

File: signUp.html (hunk 2 above)
Comment (aamrapaliwandhre) [reply signUp.html]: done

**Pull request merged**